README.md

```diff
@@ -6,8 +6,8 @@
 Create marks and measures for arbitrary units of time. Measure latency of database calls, third party requests, or code blocks and visualize them in [IOpipe](https://iopipe.com)!
 
 ## Requirements
-- Node >= `4.3.2`
-- NPM >= `2.14.12`
+- Node >= `8.10.0`
```

Comment:
We should add some config/usage instructions to the README. Similar to what's there for auto http.

Reply:
Initial doc added to README.